### PR TITLE
metric: fix typo in pilot_xds_pushes

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

### CODE REVIEW

File: pilot/pkg/xds/monitoring.go [hunk 1]
```diff
@@ -84,16 +84,16 @@ var (
...
 	)
 
-	// Covers xds_builderr and xds_senderr for xds in {lds, rds, cds, eds}.
+	// Covers xds builder and xds sender for xds in {lds, rds, cds, eds}.
 	pushes = monitoring.NewSum(
 		"pilot_xds_pushes",
 		"Pilot build and send errors for lds, rds, cds and eds.",
 	)
 
-	cdsSendErrPushes = pushes.With(typeTag.Value("cds_senderr"))
```
Comment (ramaraochavali) [inline pilot/pkg/xds/monitoring.go:93]: The existing name is correct. It is short type of "send errors"

Comment (ramaraochavali) [reply pilot/pkg/xds/monitoring.go:93]: it would be a breaking change for folks who already use it in their dashboards/alerts etc. IMO it is not worth.

Comment (howardjohn) [reply pilot/pkg/xds/monitoring.go:93]: Yes it is is "send err". We shouldn't change it IMO, at least not in isolation. In https://docs.google.com/document/d/1e86R7IoXVv_xFge_2iyI52f6m3BmKzBcJzf90fqd0ps/edit?usp=drive_open&ouid=109423391226753985670&resourcekey=0-X_jPaC7voBavDZhmoDxmdg I am thinking of a metrics re-haul, we shouldn't make isolated breaking changes

Comment (zirain) [reply pilot/pkg/xds/monitoring.go:93]: ok, make sense for now.

Comment (zirain) [reply pilot/pkg/xds/monitoring.go:93]: > Yes it is is "send err". We shouldn't change it IMO, at least not in isolation. In https://docs.google.com/document/d/1ihvByEJiWy5grWsacc6KoWBPfJ4Db9Db6yP8Lic5UXU/edit I am thinking of a metrics re-haul, we shouldn't make isolated breaking changes

I'm thinking about normalize metric name and tag, e.g. there's a lot counter metrics contains `total`, some at then, and some in the middle of the name.

Comment (howardjohn) [reply pilot/pkg/xds/monitoring.go:93]: Yes I think we should do that but with consistency -- looking at all the metrics and defining a path forward (keep as is, deprecate, replace with a new one) and metrics stability levels, etc.

Our metrics currently are pretty bad, its more issues than just `total` or not

**Pull request closed without merging**